WORKSPACE

```diff
@@ -120,6 +120,15 @@ git_repository(
 load(
     "@io_bazel_rules_docker//docker:docker.bzl",
     "docker_repositories",
+    "docker_pull",
+)
+
+# Used to generate java ca certs.
+docker_pull(
+    name = "debian8",
+    digest = "sha256:6d381d0bf292e31291136cff03b3209eb40ef6342fb790483fa1b9d3af84ae46",
```

Comment:
I've started tagging these with comments indicating what tag I got them from and when.